api/next.pb.txt

```diff
@@ -62,7 +62,6 @@ file {
     java_package: "com.google.protobuf"
     java_outer_classname: "FieldMaskProto"
     java_multiple_files: true
-    go_package: "google.golang.org/genproto/protobuf/field_mask;field_mask"
```

Comment:
This line was probably updated because I used the latest protobuild.

Reply:
This looks like an upstream change.

Reply:
Make sure you are on the latest protobuf.

Reply:
Ok, so I discovered some missing deps. I'll take of this in another PR.